client/reader/sidebar/index.jsx

```diff
@@ -295,7 +295,7 @@ export class ReaderSidebar extends Component {
 			path: this.props.path,
 			onClick: this.handleClick,
 			requireBackLink: true,
-			backLinkText: i18n.translate( 'All sites' ),
+			siteTitle: i18n.translate( 'Reader' ),
```

Comment:
nit: a personal preference on `siteTitle`, was thinking this seems more like a `pageTitle` to me.

Reply:
hmmm yea probably but wouldn't pageTitle be like the "search" or "discover" page in reader? it's kind of a hard one to name. But then again "Profile" in only one page so that would just be the page title.

Reply:
That's a good point!
Would `sideTitle` be any better, it just popped up randomly in my head this second while clicking on the heart emoji 😅 
Anyways, this isn't a blocker for merging 👍

Reply:
haha i don't know, reading it again probably `pageTitle` makes more sense I think? I might just change to use that.